src/lib.rs

```diff
@@ -114,6 +112,8 @@ impl DrawState {
         self.stencil = Some(state::Stencil {
             front: side,
             back: side,
+            front_ref_value: value,
```

Comment:
This is a bit rough. Perhaps, we should move them even higher in the spirit of this PR?